107/src/main/java/org/ehcache/jsr107/ConfigurationMerger.java

```diff
@@ -111,13 +111,13 @@
 
       CacheConfiguration<K, V> cacheConfiguration = builder.build();
 
-      if (!useJsr107Expiry) {
+      if (hasConfiguredExpiry) {
```

Comment:
Why changing the test from `(!useJsr107Expiry)` to `(hasConfiguredExpiry)` instead of `(!hasConfiguredExpiry)`? An overlook maybe?